app/controllers/concerns/circuit_breaker.rb

```diff
@@ -4,7 +4,10 @@ class NotAvailablePathMissingError < RuntimeError; end
   extend ActiveSupport::Concern
 
   included do
-    rescue_from GetIntoTeachingApiClient::CircuitBrokenError, with: :redirect_to_not_available
+    rescue_from GetIntoTeachingApiClient::CircuitBrokenError do |exception|
+      Sentry.capture_exception(exception)
```

Comment:
Can we update the spec to `expect(Sentry).to receive(:capture_exception).with(exception)`?